### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

Added new juju.is homepage, minus "what is juju" section (being worked on separately)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8041
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches [the design](https://app.zeplin.io/project/606203889af57e12287b5308/screen/606edfa32c8c0b3ac8935e2c) and [the copy doc](https://docs.google.com/document/d/147spDkSVH0YaEGKZ4tDi7i28zzhIcacF1hBOMHvxaC0/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2020

### CODE REVIEW

File: static/sass/_patterns_carousel.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
+@mixin juju-carousel {
+  .p-carousel {
+    position: relative;
+
+    .p-carousel__cell {
+      // height: 100%;
```
Comment (solazio) [inline static/sass/_patterns_carousel.scss:6]: I guess you can remove the commented out code

File: templates/partials/_contributors.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  <div class="row">
+    <div class="col-12">
+      <h2>Users and contributors</h2>
+    </div>
+
+    <div class="col-12">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            <img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/89680bdf-logo-ubuntu.svg" alt="Ubuntu logo">
```
Comment (solazio) [inline templates/partials/_contributors.html:11]: can you use the image template instead, for all these images?

File: templates/partials/_juju-supports.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
+<section class="p-strip">
+  <div class="row">
```
Comment (solazio) [inline templates/partials/_juju-supports.html:2]: any reason you use `row>col-12` instead of `u-fixed-width`?

**Review by solazio — APPROVED** [review]
Looks good, just a few comments in the code. Thanks

File: templates/partials/_contributors.html (hunk 2 above)
Comment (sowasred2012) [reply templates/partials/_contributors.html:11]: Yep - when we have the correct images and can get their heights/widths, for this I didn't want to spend time doing that for placeholder images.

File: static/sass/_patterns_carousel.scss (hunk 1 above)
Comment (sowasred2012) [reply static/sass/_patterns_carousel.scss:6]: :+1:

File: templates/partials/_juju-supports.html (hunk 3 above)
Comment (sowasred2012) [reply templates/partials/_juju-supports.html:2]: Personal preference - when I'm using nested rows/columns I like that it's clear from the markup how many columns the parent has.

**Review by solazio — APPROVED** [review]

**Pull request merged**